clcache.py

```diff
@@ -206,7 +206,7 @@ def computeKey(compilerBinary, commandLine):
         (preprocessedSourceCode, pperr) = preprocessor.communicate()
 
         if preprocessor.returncode != 0:
-            sys.stderr.write(pperr)
+            sys.stderr.write(pperr.decode("utf-8"))
```

Comment:
How can we know that cl's output is UTF-8? As far as I know it is not and there is proper documentation missing about cl's encoding.